googlemock/CMakeLists.txt

```diff
@@ -78,6 +78,11 @@ set(gmock_build_include_dirs
   "${gtest_SOURCE_DIR}")
 include_directories(${gmock_build_include_dirs})
 
+# Set minimum C++ version to C++11
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
If we accept this change, how would I build GoogleTest if I wanted C++20 support?

Reply:
The other pull request that just got merged is better.  I will close this.